### PR TITLE
added the worlds holidays

### PR DESCRIPTION
**Pull request by petpetti**
I add the many holidays of the world , includes September and june .

this very useful for plugin that need the holidays , september, october, november , and summer

you can now do 

$this->getholiday()1 //june
$this->getholiday()2 //april

and add the holidays too !

please test

### CODE REVIEW

**Comment by petpetti** [comment]
@meu32ki it is nessaracy, it adds very useful api methods. for example

if($this->holiday === "christmas){
//send message merry christmass!

it helps chunk performance too when you use clear holiday

File: src/pocketmine/Player.php [hunk 1]
```diff
@@ -1,55 +1,3811 @@
-<?php
+
+use pocketmine\event\entity\EntityDamageEvent;
+use pocketmin<?php
```
Comment (Leo3418) [inline src/pocketmine/Player.php]: Did you check this?

**Comment by Leo3418** [comment]
The API methods you proposed might only be necessary to YOU. If you really think it's necessary to everyone, you can write a plugin which implements holiday and provides API to other plugins, then publish it.

**Comment by petpetti** [comment]
@Leo3418 im so sorry :(

maybe i can fix it and reopen ?

**Comment by Leo3418** [comment]
I don't think the GenisysPro dev team and I will agree that this feature is useful to most of our users, so my apology, this pull request may not be reopened.

**Comment by meu32ki** [comment]
@petpetti Okey, if you think it's useful, thats fine. But in the way of coding, it's "not beautiful" as the holidays are no connections to player. I mean, Those function should not be in Player.php (should I say  Class::Player maybe?). As Leo says, it would be better for everyone to write them in plugin which gives APIs and useful functions.

Sorry Leo, but he needs to know how PMMP works I believe.

**Pull request closed without merging**